core/shared/platform/windows/win_socket.c

```diff
@@ -6,6 +6,8 @@
 #include "platform_api_vmcore.h"
 #include "platform_api_extension.h"
 
+#include <ws2tcpip.h>
```

Comment:
The header file has already been included in "core/shared/platform/windows/platform_internal.h", which is then included in "platform_api_vmcore.h", wondering why there is build issue? What is the error reported?

Reply:
oh, you're right. I was getting errors about `inet_pton` not being found, but that must have been some weirdness of my build configuration. After trying again, there's no error even when the header here is not added.